pyprep/utils.py

```diff
@@ -64,3 +64,29 @@ def filter_design(N_order, amp, freq):
     )
     kernel = np.multiply(kernel[0 : N_order + 1], (np.transpose(hamming_window[:])))
     return kernel
+
+
+def SplitList(mylist, chunk_size):
```

Comment:
why call it in CamelCase with Capital first letter? according to [pep8](https://www.python.org/dev/peps/pep-0008/#naming-conventions), this should rather be `split_list`

Reply:
explanation: just my laziness